app/screens/channel/channel_post_list/channel_post_list.js

```diff
@@ -181,18 +180,15 @@ export default class ChannelPostList extends PureComponent {
             lastViewedAt,
             loadMorePostsVisible,
             refreshing,
-            theme,
         } = this.props;
 
         const {visiblePostIds} = this.state;
         let component;
 
         if (visiblePostIds.length === 0 && channelRefreshingFailed) {
+            const Loading = require('app/components/channel_loader').default;
             component = (
-                <PostListRetry
-                    retry={this.loadPostsRetry}
-                    theme={theme}
-                />
+                <Loading channelIsLoading={true}/>
```

Comment:
Why was the PostListRetry removed from here? in case this fails it will show the channel loader forever instead of providing the user with more feedback

Reply:
Because we already have a bar at the top that shows the user the information.
The post list retry is not showing any information, just a refresh button.

Reply:
yeah, but I rather not have an endless channel loading, to me that is as bad as the previous one.. Maybe show the cloud image with some text or something other than an endless channel loader

Reply:
@enahum We have another larger ticket to look at the different states.
I can ask Andrew to look into the design and the message that needs to be there for it.
https://mattermost.atlassian.net/browse/MM-16251